### DISCUSSION TITLE
Support Older GLibc 2.26 (Amazon Linux 2)

### DISCUSSION THREAD

**Discussion by AlJohri** [aws/codewhisperer-command-line-discussions#195] [Support ticket]
### Checks

- [X] I ran `q doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `q restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

I am trying to setup Amazon Q using ssh my remote machine (cloud desktop) via these instructions: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html

### Actual behaviour

I run into the following error when I run the `install.sh` script:

```
❯ ./q/install.sh
/home/aljohri/.local/bin/q: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.27' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.29' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by /home/aljohri/.local/bin/q)
/home/aljohri/.local/bin/q: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /home/aljohri/.local/bin/q)
```

### Steps to reproduce

Try to install Amazon Q on any Amazon Linux 2 machine.

### Environment

```shell
❯ cat /etc/system-release
Amazon Linux release 2 (Karoo)

❯ cat /etc/os-release
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2:-:internal"
HOME_URL="https://amazonlinux.com/"
VARIANT="internal"

❯ ldd --version
ldd (GNU libc) 2.26
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

**Comment by grant0417** [comment]
We are working on bring support for the CLI to AL2 and non-glibc based distros. We expect to support these very soon but no exact date yet.

**Reply by AlJohri** [reply]
Hey @grant0417, any updates on when this might be supported?

**Reply by grant0417** [reply]
I have messaged you the solution for Amazon employees
